webpack/dev.js

```diff
@@ -14,6 +14,7 @@ const dev = webpackMerge(base, {
     port: 8080,
     host: 'localhost',
     contentBase: buildPath,
+    historyApiFallback: true,
```

Comment:
This may cause a merge conflict. GitHub should warn you if this is the case

Reply:
Sure, will look out for that.